### PR TITLE
Convert docs from MarkDown to AsciiDoc

### PR DESCRIPTION
**Pull request by fao89**
Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1098
Preview: https://logserver.rdoproject.org/32/632/089782abf464edf0ed8c63c2b62ad9d5089dd1a9/github-check/dataplane-operator-docs-preview/74a947a/docs_build/docs_build/dataplane/index-upstream.html

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -76,6 +76,31 @@ SHELL = /usr/bin/env bash -o pipefail
...
+		echo "Bundler not found. On Linux run 'sudo dnf install /usr/bin/bundle' to install it."; \
+		exit 1; \
+	fi
+
+	bundle config set --local path 'local/bundle'; bundle install
+
+.PHONY: docs
+docs: manifests docs-dependencies crd-to-markdown ## Build docs
+	$(CRD_MARKDOWN) -f api/v1beta1/common.go -f api/v1beta1/openstackdataplaneservice_types.go -f api/v1beta1/openstackdataplanenodeset_types.go -f api/v1beta1/openstackdataplanedeployment_types.go -n OpenStackDataPlaneService -n OpenStackDataPlaneNodeSet -n OpenStackDataPlaneDeployment > docs/assemblies/custom_resources.md
+	bundle exec kramdoc --auto-ids docs/assemblies/custom_resources.md && rm docs/assemblies/custom_resources.md
```
Comment (fao89) [inline Makefile:96]: converting md to adoc

**Comment by fao89** [comment]
recheck

**Review by slagle — APPROVED** [review]

**Pull request merged**